internal/delivery/http/route_boards_articles_test.go

```diff
@@ -0,0 +1,42 @@
+package http
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+)
+
+func TestGetBoardArticlesBadRequest(t *testing.T) {
+	userID := "id"
+	usecase := NewMockUsecase()
+	delivery := NewHTTPDelivery(usecase)
+
+	title := ""
+	author := ""
+	recommendCountGe := "qwerty"
```

Comment:
```suggestion
	recommendCountGreaterEqual := "qwerty"
```

建議在縮寫的部分是改成 GE 或者是 GreaterEqual

Reply:
這部分的 code 是我拉 development 下來的部分
不在這次修改範圍